test/cmd/test-scenario/generator/generators_test.go

```diff
@@ -1,4 +1,4 @@
-package main
+package generator
```

Comment:
```suggestion
package generator_test
```

Isn't it the go way?

Can't do that in the suggestion but you will have to fix imports too.

Reply:
https://github.com/Maistra/istio-workspace/pull/386/commits/c2fc1bdc0b05a205335c3e8db671727fcada2b9d